.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Historically we haven't committed IDE configs. I can see a case for it, but probably best discussed in a separate issue/PR?

Reply:
Didn't mean to add this I will remove it.

Reply:
Can I add .vscode the git ignore, so this does not happen again?

Reply:
sure